diff --git a/src/parameters.ml b/src/parameters.ml
index 440a3a3..fc34af5 100644
--- a/src/parameters.ml
+++ b/src/parameters.ml
@@ -224,6 +224,7 @@ let[@inline] compute_current_burrow_fee_index (last_burrow_fee_index: fixedpoint
 *)
 let[@inline] compute_current_protected_index (last_protected_index: Ligo.nat) (current_index: Ligo.nat) (duration_in_seconds: Ligo.int) : Ligo.nat =
   assert (Ligo.gt_nat_nat last_protected_index (Ligo.nat_from_literal "0n"));
+  let duration_in_seconds = Ligo.mul_int_int duration_in_seconds (Ligo.int_from_literal "1000") in
   let last_protected_index = Ligo.int last_protected_index in
   fraction_to_nat_floor
     (clamp_int
