--- ./src/constants.ml
+++ ./src/constants.ml
@@ -43,9 +43,9 @@

 let[@inline] auction_decay_rate : ratio = make_ratio (Ligo.int_from_literal "1") (Ligo.int_from_literal "6000")

-let[@inline] max_bid_interval_in_seconds : Ligo.int = Ligo.int_from_literal "1200"
+let[@inline] max_bid_interval_in_seconds : Ligo.int = Ligo.int_from_literal "1"

-let[@inline] max_bid_interval_in_blocks : Ligo.nat = Ligo.nat_from_literal "20n"
+let[@inline] max_bid_interval_in_blocks : Ligo.nat = Ligo.nat_from_literal "1n"

 let[@inline] bid_improvement_factor : ratio = make_ratio (Ligo.int_from_literal "33") (Ligo.int_from_literal "10000")

diff --git a/src/parameters.ml b/src/parameters.ml
index e53b279..915f26a 100644
--- src/parameters.ml
+++ src/parameters.ml
@@ -224,6 +224,7 @@ let[@inline] compute_current_burrow_fee_index (last_burrow_fee_index: fixedpoint
 *)
 let[@inline] compute_current_protected_index (last_protected_index: Ligo.nat) (current_index: Ligo.nat) (duration_in_seconds: Ligo.int) : Ligo.nat =
   assert (Ligo.gt_nat_nat last_protected_index (Ligo.nat_from_literal "0n"));
+  let duration_in_seconds = Ligo.mul_int_int duration_in_seconds (Ligo.int_from_literal "1000") in
   fraction_to_nat_floor
     (clamp_int
        (Ligo.mul_nat_int
