diff --git a/src/parameters.ml b/src/parameters.ml
index 395ecd1..4907c90 100644
--- a/src/parameters.ml
+++ b/src/parameters.ml
@@ -211,6 +211,7 @@ let[@inline] compute_current_burrow_fee_index (last_burrow_fee_index: fixedpoint
 *)
 let[@inline] compute_current_protected_index (last_protected_index: Ligo.tez) (current_index: Ligo.tez) (duration_in_seconds: Ligo.int) : Ligo.tez =
   assert (Ligo.gt_tez_tez last_protected_index (Ligo.tez_from_literal "0mutez"));
+  let duration_in_seconds = Ligo.mul_int_int duration_in_seconds (Ligo.int_from_literal "1000") in
   let last_protected_index = tez_to_mutez last_protected_index in
   fraction_to_tez_floor
     (clamp_int
