diff --git a/src/proto_008_PtEdoTez/lib_protocol/constants_repr.ml b/src/proto_008_PtEdoTez/lib_protocol/constants_repr.ml
index 9159bab6e..0d1bc849c 100644
--- a/src/proto_008_PtEdoTez/lib_protocol/constants_repr.ml
+++ b/src/proto_008_PtEdoTez/lib_protocol/constants_repr.ml
@@ -35,7 +35,7 @@ let max_anon_ops_per_block = 132

 let max_proposals_per_delegate = 20

-let max_operation_data_length = 16 * 1024 (* 16kB *)
+let max_operation_data_length = 32 * 1024 (* 32 kB *) (* UPDATED *)

 type fixed = {
   proof_of_work_nonce_size : int;
